### DISCUSSION TITLE
Add new city request: Oldenburg

### DISCUSSION THREAD

**Discussion by larsschwarz** [EqualStreetNames/equalstreetnames#762] [New city]
Hey guys,

updated the template according to the docs:

https://github.com/larsschwarz/equalstreetnames-template/tree/master

Thanks: Lars

**Comment by jbelien** [comment]
Hello @larsschwarz ,

Welcome to the project!
You should have received an invite to join @EqualStreetNames/oldenburg ; once you've joined you will be able to transfer your repository to @EqualStreetNames

**Don't forget to rename your repository to `equalstreetnames-oldenburg` before transfer!**

**Comment by larsschwarz** [comment]
Hey @jbelien, 

Thanks! Renamed and transferred.

**Comment by jbelien** [comment]
Everything is ready on my side, you just need to add `MAPBOX_TOKEN` secret in your repo.

**Comment by larsschwarz** [comment]
Added!

**Comment by jbelien** [comment]
🚀 Deployed: https://oldenburg.equalstreetnames.eu/

**Comment by larsschwarz** [comment]
Still waiting for DNS propagation, but: Thanks Jonathan!
